docs/tornjak-quickstart.md

```diff
@@ -261,6 +267,137 @@ Note that there are four key differences in this StatefulSet file from that in t
 
 This is all done specifically to pass the Tornjak config file as an argument to the container. 
 
+</details>
+
+<details><summary>[Click] For the Tornjak-backend sidecar implementation</summary>
+
+There is an additional requirement to mount the SPIRE server socket and make it accessible to the Tornjak backend container. 
+
+The statefulset will look something like this, where we have commented leading with a 👈 on the changed or new lines: 
+
+```
+➜  quickstart git:(master) cat server-statefulset.yaml 
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: spire-server
+  namespace: spire
+  labels:
+    app: spire-server
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: spire-server
+  serviceName: spire-server
+  template:
+    metadata:
+      namespace: spire
+      labels:
+        app: spire-server
+    spec:
+      serviceAccountName: spire-server
+      containers:
+        - name: spire-server
+          image: ghcr.io/spiffe/spire-server:1.4.4
+          args:
+            - -config
+            - /run/spire/config/server.conf
+          ports:
+            - containerPort: 8081
+          volumeMounts:
+            - name: spire-config
+              mountPath: /run/spire/config
+              readOnly: true
+            - name: spire-data
+              mountPath: /run/spire/data
+              readOnly: false
+            - name: test-socket # 👈 ADDITIONAL VOLUME
+              mountPath: /tmp/spire-server/private # 👈 ADDITIONAL VOLUME
+          livenessProbe:
+            httpGet:
+              path: /live
+              port: 8080
+            failureThreshold: 2
+            initialDelaySeconds: 15
+            periodSeconds: 60
+            timeoutSeconds: 3
+          readinessProbe:
+            httpGet:
+              path: /ready
+              port: 8080
+            initialDelaySeconds: 5
+            periodSeconds: 5
+	### 👈 BEGIN ADDITIONAL CONTAINER ###
+        - name: tornjak-backend
+          image: docker.io/tsidentity/tornjak-be:latest
+          args:
+            - -config
+            - /run/spire/config/server.conf
+            - -tornjak-config
+            - /run/spire/tornjak-config/server.conf
+          ports:
+            - containerPort: 8081
+          volumeMounts:
+            - name: spire-config
+              mountPath: /run/spire/config
+              readOnly: true
+            - name: tornjak-config
+              mountPath: /run/spire/tornjak-config
+              readOnly: true
+            - name: spire-data
+              mountPath: /run/spire/data
+              readOnly: false
+            - name: test-socket
+              mountPath: /tmp/spire-server/private
+          livenessProbe:
```

Comment:
since Tornjak runs on port 8081, should the `livenessProbe` and `readinessProbe` measure the Tornjak service on port 8081? I think this part needs further investigation